oe_theme.theme

```diff
@@ -668,6 +668,19 @@ function oe_theme_preprocess_pattern_accordion(array &$variables): void {
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_banner_video(array &$variables): void {
```

Comment:
We should add in this function the way to change a single source to html5.
we change the video field of banner_video. we should create an 'upgrade path'.

Reply:
In this case there is no upgrade path as such. There is no actual implementation of any banner so there is nothing to update but the pattern. If there were components out there using this pattern they would need to deal with the update themselves since we are deealing with a major upgrade.

Reply:
Ok I agree.
thanks